.golangci.yml

```diff
@@ -179,6 +179,14 @@ issues:
       linters:
         - revive
 
+    - text: "package-comments:"
+      linters:
+        - revive
+    
+    - text: "exported:"
+      linters:
+        - revive
+
```

Comment:
Why should we add this?

Reply:
The address linting errors, in [ci lint](https://github.com/kubevela/terraform-controller/actions/runs/3560027171/jobs/5979782813#step:4:86)
```
Error: controllers/configuration/configuration.go:1:1: package-comments: should have a package comment (revive)
```

```
Error: api/types/state.go:43:2: exported: exported const TerraformInit should have comment (or a comment on this block) or be unexported (revive)
```

Reply:
OK, got it.